src/styles/blocks/calendar.scss

```
@@ -0,0 +1,49 @@
+@import "../utils/variables";
+
+.calendar {
+  display: flex;
+  gap: 1px;
+  max-width: $day-form * 7;
+  flex-wrap: wrap;
+  min-width: 800px;
+
+  &__day {
+    display: flex;
+    width: 100px;
+    height: 100px;
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: 0.5s;
```

transition потрібно встановлювати на стилі які анімуються, якщо цього не робити то transition буде становлючатись на всі стилі
```suggestion
    transition: transform  0.5s;
```